### DISCUSSION TITLE
Dark mode is enabled, but is not working

### DISCUSSION THREAD

**Discussion by UCYT5040** [tailwindlabs/tailwindcss#6081] [Help]
I am using Create React Project.

Everything seems to work, but dark mode does not.

Code:
```js
// tailwind.config.js

module.exports = {
    purge: [],
    darkMode: 'media', // or false or 'class'
    theme: {
      extend: {},
    },
    variants: {
      extend: {
        display: ['dark']
      },
    },
    plugins: [],
  }
```
```
<div className="dark:text-red-500 text-blue-400">
      hello
</div>
```
^ This gives me blue words not red

Here is a file structure screenshot:
![image](https://user-images.githubusercontent.com/46662314/141664408-63f4530c-b3b3-4e4a-b29b-b2f7f7caded0.png)

**Comment by NovusOpus** [comment]
Do you have Dark Mode Enabled in your System? 

Quote from Tailwind CSS Docs:

> To enable it, set the darkMode option in your tailwind.config.js file to media:
> 
> ```JS
> // tailwind.config.js
> module.exports = {
>   darkMode: 'media',
>   // ...
> }
> ```
> Now whenever **dark mode is enabled on the user’s operating system**, `dark:{class} `classes will take precedence over unprefixed classes. The media strategy uses the prefers-color-scheme media feature under the hood, but if you’d like to support toggling dark mode manually, you can also use the ‘class’ strategy for more control.

To see the dark mode your system have to be setup on dark mode, try changing your settings in your os to dark mode if you don't have it already.

Also here's a working example: [https://play.tailwindcss.com/qPE1l0bqbD](https://play.tailwindcss.com/qPE1l0bqbD)(Tested and working)
